apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-docs.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component } from '@angular/core';
+
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+
+import * as dynamicPageBasicExample from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-example.component.html';
+
+import * as dynamicPageTabbedExample from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-tabbed-example.component.html';
+import * as dynamicPageTabbedExampleScss from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-tabbed-example.component.scss';
+import * as dynamicPageTabbedExampleTsCode from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-tabbed-example.component.ts';
+
+import * as dynamicPageSnapScrollExample from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-snap-scroll-example.component.html';
+import * as dynamicPageSnapScrollExampleScss from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-snap-scroll-example.component.scss';
+import * as dynamicPageSnapScrollExampleTsCode from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-snap-scroll-example.component.ts';
+
+import * as dynamicPageResponsivePaddingExample from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-responsive-padding-example.component.html';
+import * as dynamicPageResponsivePaddingExampleTsCode from '!raw-loader!./platform-dynamic-page-examples/platform-dynamic-page-responsive-padding-example.component.ts';
+
+@Component({
+    selector: 'app-dynamic-page',
+    templateUrl: './platform-dynamic-page-docs.component.html'
+})
+export class PlatformDynamicPageDocsComponent {
+    dynamicPageBasic: ExampleFile[] = [
+        {
+            language: 'html',
+            code: dynamicPageBasicExample,
+            fileName: 'platform-dynamic-page-example'
+        }
```

Comment:
I think you would need to add ts file there as well, because there are used methods in template

Reply:
added ts to this example and css files to other examples. Removed a redundant css file.